login.css

```diff
@@ -0,0 +1,90 @@
+@import url("https://peaceui.netlify.app/peace.css");
+
+/* overriding btn component */
+.btn{
+    margin: 0;
+}
+
+.btn-ghost:hover{
+    color: var(--primary-100);
+    border: 2px solid var(--primary-100);
+}
+
+.btn-ghost:active{
+    color: var(--primary);
+}
+
+/* overriding navbar */
+.nav-wrapper{
+    top: 0;
+    width: 100%;
+}
+
+.nav-list{
+    /* reduced column-gap from 3em to 1em */
+    column-gap: 1em;
+    margin-right: 0;
+}
+
+/* overriding input */
+.search{
+    margin:0;
+    padding: 0;
+    height: 1rem;
+    display: block;
+    cursor: pointer;
+}
+
+.input-data{
+    border: 2px solid var(--background-grey);
+}
+
+.input-data:hover{
+    border: 2px solid var(--primary);
+}
+
+/* overriding badge */
+.badge{
+    margin:0;
+}
+
+.icon-badge > i{
+    font-size: 1.8em;
+}
+
+/* login section */
+
+.login-area{
+    display: flex;
+    justify-content: center;
+    width: 100%;
+}
+
+form{
```

Comment:
Suggestion - Can use a class selector, so that your other forms are not changed.